### ISSUE TITLE
Installation fails on 0.32-beta

### ISSUE THREAD

**Issue by spantaleev** [zadam/trilium#537]
On a brand new instance using the `0.32-latest` Docker image (`docker run -it --rm -p 8080:8080 -v ~/trilium-data:/root/trilium-data zadam/trilium:0.32-latest`), trying to install by creating a new user from the web interface yields the following:

```
Generated session secret
Generated sourceId=fxYYPhUhDKx5
App HTTP server starting up at port 8080
{
  "appVersion": "0.32.0-beta",
  "dbVersion": 136,
  "syncVersion": 9,
  "buildDate": "2019-05-22T22:28:35+02:00",
  "buildRevision": "35648b9f3756294f91809ed6e5dbc3a30c7c9f13",
  "dataDirectory": "/root/trilium-data"
}
DB not initialized, please visit setup page to see instructions on how to initialize Trilium.


Creating initial database ...
One of the tagRootNote, doneRootNote or todoRootNote does not exist
One of the tagRootNote, doneRootNote or todoRootNote does not exist
One of the tagRootNote, doneRootNote or todoRootNote does not exist
One of the tagRootNote, doneRootNote or todoRootNote does not exist
Creating missing sync record for notes 0UdTNH7HpJQc
Creating missing sync record for notes 0nZDX3WJ5gcX
Creating missing sync record for notes 1BcxJmwvrt9G
Creating missing sync record for notes 1Bg4qzg8sm3I
Creating missing sync record for notes 1bqvyMUfzTKk
Creating missing sync record for notes 2v0QIZiZuprl
Creating missing sync record for notes 3HXAxpXziOQs
Creating missing sync record for notes 3QCBWiXh0VVj
Creating missing sync record for notes 78NBT9Tu82Sy
Creating missing sync record for notes 7F0TGCSHxYpS
Creating missing sync record for notes 7HysEmtqsmRf
Creating missing sync record for notes 7e7cGOEJlRiW
Creating missing sync record for notes 8YY712jB6FE2
Creating missing sync record for notes 96BaAuCxXbfJ
Creating missing sync record for notes A8hWkgnj6fbh
Creating missing sync record for notes AY2YbnBLxFqA
Creating missing sync record for notes AhBA341X5rc8
Creating missing sync record for notes B0vCkpIjbmr2
Creating missing sync record for notes CQvKPco1sGlF
Creating missing sync record for notes CUUj2PtEGM0q
Creating missing sync record for notes CwJ64Ns56FWm
Creating missing sync record for notes CzUEqUePixXe
Creating missing sync record for notes DL33MlOegNVw
Creating missing sync record for notes DjI8B3CoSlVr
Creating missing sync record for notes E4SbClgwsmEo
Creating missing sync record for notes E4wF6OxTnf6u
Creating missing sync record for notes E6mArRDG4fFJ
Creating missing sync record for notes Ebr0yzP50YsQ
Creating missing sync record for notes FDE3qWrGpMwD
Creating missing sync record for notes FIDYYXaUNbqd
Creating missing sync record for notes FIsvTshVgf9R
Creating missing sync record for notes FReug62oB19d
Creating missing sync record for notes GJSAgqVdGsvf
Creating missing sync record for notes H4fYXdwST988
Creating missing sync record for notes HxzQvbBKha6b
Creating missing sync record for notes J1gxzhFj87Fi
Creating missing sync record for notes JbTRBm3sg7Dw
Creating missing sync record for notes KdIahTNCFXcy
Creating missing sync record for notes KvjDVO2Z9qSN
Creating missing sync record for notes LRIwIyzlM25o
Creating missing sync record for notes NGYYyhFX0sBs
Creating missing sync record for notes NaiLy1NZhWWg
Creating missing sync record for notes O337sHuTWijR
Creating missing sync record for notes OIDPZhQs99Rd
Creating missing sync record for notes PXCSQJVwGZk1
Creating missing sync record for notes Privena1gpI4
Creating missing sync record for notes QYXF6PoJy8cR
Creating missing sync record for notes RvpkZKqMwzw1
Creating missing sync record for notes SHrRG393Eeek
Creating missing sync record for notes SLeiYjzKUQZc
Creating missing sync record for notes SWYgWgWFLCJf
Creating missing sync record for notes TWE9jh8913tN
Creating missing sync record for notes U297yHLcMPrI
Creating missing sync record for notes UBXu4YyRekO4
Creating missing sync record for notes UMscyfzKknx2
Creating missing sync record for notes V2QKOK3hUUhn
Creating missing sync record for notes VeRW2CFhPDnf
Creating missing sync record for notes WH4XmEPixVFy
Creating missing sync record for notes WT4Nk4uYtWBB
Creating missing sync record for notes WbU7xPcskLBf
Creating missing sync record for notes Wus4NDfYsw2A
Creating missing sync record for notes XSEvE2DX022A
Creating missing sync record for notes Y8iisT5qVemX
Creating missing sync record for notes ZGbRIGTPTSiB
Creating missing sync record for notes a3UxdNUIs7Tl
Creating missing sync record for notes asEQuFSQzRGa
Creating missing sync record for notes bMz6kf1G0KXN
Creating missing sync record for notes bsBTq2kmjrcd
Creating missing sync record for notes coAj8qH7aWth
Creating missing sync record for notes dUuhy86lWWHq
Creating missing sync record for notes eFA5HLATc661
Creating missing sync record for notes eRqoNhiexEwG
Creating missing sync record for notes esd0tQhjnNjp
Creating missing sync record for notes evyVAjox8iuB
Creating missing sync record for notes fiq8sI65RoGV
Creating missing sync record for notes gV8Hhgax3lkp
Creating missing sync record for notes h1iYm0ObpZMq
Creating missing sync record for notes hGZtvHujVJOP
Creating missing sync record for notes iAykSnIlvldC
Creating missing sync record for notes jFBSlDOpJlyx
Creating missing sync record for notes jeVTtlcQdEGB
Creating missing sync record for notes kG7I400CPAnv
Creating missing sync record for notes m86sLM0ZpCfI
Creating missing sync record for notes mIT422wbVYIO
Creating missing sync record for notes mTyhRETTAlep
Creating missing sync record for notes mZU606wuVjv9
Creating missing sync record for notes n9H5eT1lhfgc
Creating missing sync record for notes nSa98Jj5kCzb
Creating missing sync record for notes r0OGAzFDYizJ
Creating missing sync record for notes r8t2M8hAC3I6
Creating missing sync record for notes rCMlGVuAsZ0q
Creating missing sync record for notes rgfydwTOGoYu
Creating missing sync record for notes root
Creating missing sync record for notes sKfgQIZREtf5
Creating missing sync record for notes tYPP67zSJ8tk
Creating missing sync record for notes wQQWrQfgj2Uq
Creating missing sync record for notes wcO4JmhNOf4w
Creating missing sync record for notes wxFyBJTG7BCD
Creating missing sync record for notes xhwLPRDWQa1N
Creating missing sync record for notes yRbM42icFvWz
Creating missing sync record for notes zGdTQKhm9IHj
Creating missing sync record for note_contents 0UdTNH7HpJQc
Creating missing sync record for note_contents 0nZDX3WJ5gcX
Creating missing sync record for note_contents 1BcxJmwvrt9G
Creating missing sync record for note_contents 1Bg4qzg8sm3I
Creating missing sync record for note_contents 1bqvyMUfzTKk
Creating missing sync record for note_contents 2v0QIZiZuprl
Creating missing sync record for note_contents 3HXAxpXziOQs
Creating missing sync record for note_contents 3QCBWiXh0VVj
Creating missing sync record for note_contents 78NBT9Tu82Sy
Creating missing sync record for note_contents 7F0TGCSHxYpS
Creating missing sync record for note_contents 7HysEmtqsmRf
Creating missing sync record for note_contents 7e7cGOEJlRiW
Creating missing sync record for note_contents 8YY712jB6FE2
Creating missing sync record for note_contents 96BaAuCxXbfJ
Creating missing sync record for note_contents A8hWkgnj6fbh
Creating missing sync record for note_contents AY2YbnBLxFqA
Creating missing sync record for note_contents AhBA341X5rc8
Creating missing sync record for note_contents B0vCkpIjbmr2
Creating missing sync record for note_contents CQvKPco1sGlF
Creating missing sync record for note_contents CUUj2PtEGM0q
Creating missing sync record for note_contents CwJ64Ns56FWm
Creating missing sync record for note_contents CzUEqUePixXe
Creating missing sync record for note_contents DL33MlOegNVw
Creating missing sync record for note_contents DjI8B3CoSlVr
Creating missing sync record for note_contents E4SbClgwsmEo
Creating missing sync record for note_contents E4wF6OxTnf6u
Creating missing sync record for note_contents E6mArRDG4fFJ
Creating missing sync record for note_contents Ebr0yzP50YsQ
Creating missing sync record for note_contents FDE3qWrGpMwD
Creating missing sync record for note_contents FIDYYXaUNbqd
Creating missing sync record for note_contents FIsvTshVgf9R
Creating missing sync record for note_contents FReug62oB19d
Creating missing sync record for note_contents GJSAgqVdGsvf
Creating missing sync record for note_contents H4fYXdwST988
Creating missing sync record for note_contents HxzQvbBKha6b
Creating missing sync record for note_contents J1gxzhFj87Fi
Creating missing sync record for note_contents JbTRBm3sg7Dw
Creating missing sync record for note_contents KdIahTNCFXcy
Creating missing sync record for note_contents KvjDVO2Z9qSN
Creating missing sync record for note_contents LRIwIyzlM25o
Creating missing sync record for note_contents NGYYyhFX0sBs
Creating missing sync record for note_contents NaiLy1NZhWWg
Creating missing sync record for note_contents O337sHuTWijR
Creating missing sync record for note_contents OIDPZhQs99Rd
Creating missing sync record for note_contents PXCSQJVwGZk1
Creating missing sync record for note_contents Privena1gpI4
Creating missing sync record for note_contents QYXF6PoJy8cR
Creating missing sync record for note_contents RvpkZKqMwzw1
Creating missing sync record for note_contents SHrRG393Eeek
Creating missing sync record for note_contents SLeiYjzKUQZc
Creating missing sync record for note_contents SWYgWgWFLCJf
Creating missing sync record for note_contents TWE9jh8913tN
Creating missing sync record for note_contents U297yHLcMPrI
Creating missing sync record for note_contents UBXu4YyRekO4
Creating missing sync record for note_contents UMscyfzKknx2
Creating missing sync record for note_contents V2QKOK3hUUhn
Creating missing sync record for note_contents VeRW2CFhPDnf
Creating missing sync record for note_contents WH4XmEPixVFy
Creating missing sync record for note_contents WT4Nk4uYtWBB
Creating missing sync record for note_contents WbU7xPcskLBf
Creating missing sync record for note_contents Wus4NDfYsw2A
Creating missing sync record for note_contents XSEvE2DX022A
Creating missing sync record for note_contents Y8iisT5qVemX
Creating missing sync record for note_contents ZGbRIGTPTSiB
Creating missing sync record for note_contents a3UxdNUIs7Tl
Creating missing sync record for note_contents asEQuFSQzRGa
Creating missing sync record for note_contents bMz6kf1G0KXN
Creating missing sync record for note_contents bsBTq2kmjrcd
Creating missing sync record for note_contents coAj8qH7aWth
Creating missing sync record for note_contents dUuhy86lWWHq
Creating missing sync record for note_contents eFA5HLATc661
Creating missing sync record for note_contents eRqoNhiexEwG
Creating missing sync record for note_contents esd0tQhjnNjp
Creating missing sync record for note_contents evyVAjox8iuB
Creating missing sync record for note_contents fiq8sI65RoGV
Creating missing sync record for note_contents gV8Hhgax3lkp
Creating missing sync record for note_contents h1iYm0ObpZMq
Creating missing sync record for note_contents hGZtvHujVJOP
Creating missing sync record for note_contents iAykSnIlvldC
Creating missing sync record for note_contents jFBSlDOpJlyx
Creating missing sync record for note_contents jeVTtlcQdEGB
Creating missing sync record for note_contents kG7I400CPAnv
Creating missing sync record for note_contents m86sLM0ZpCfI
Creating missing sync record for note_contents mIT422wbVYIO
Creating missing sync record for note_contents mTyhRETTAlep
Creating missing sync record for note_contents mZU606wuVjv9
Creating missing sync record for note_contents n9H5eT1lhfgc
Creating missing sync record for note_contents nSa98Jj5kCzb
Creating missing sync record for note_contents r0OGAzFDYizJ
Creating missing sync record for note_contents r8t2M8hAC3I6
Creating missing sync record for note_contents rCMlGVuAsZ0q
Creating missing sync record for note_contents rgfydwTOGoYu
Creating missing sync record for note_contents root
Creating missing sync record for note_contents sKfgQIZREtf5
Creating missing sync record for note_contents tYPP67zSJ8tk
Creating missing sync record for note_contents wQQWrQfgj2Uq
Creating missing sync record for note_contents wcO4JmhNOf4w
Creating missing sync record for note_contents wxFyBJTG7BCD
Creating missing sync record for note_contents xhwLPRDWQa1N
Creating missing sync record for note_contents yRbM42icFvWz
Creating missing sync record for note_contents zGdTQKhm9IHj
Creating missing sync record for branches 2HrEv0XLpbcO
Creating missing sync record for branches 3O74RneJSTdm
Creating missing sync record for branches 3bGWpJoqrwvn
Creating missing sync record for branches 3fa117c5DJH6
Creating missing sync record for branches 4p4i5ZgzkjLz
Creating missing sync record for branches 4tKWaWnpcmDY
Creating missing sync record for branches 4wqYdHTPuiUN
Creating missing sync record for branches 5DkCk0VQCPT8
Creating missing sync record for branches 6Vv4dUA3CQcH
Creating missing sync record for branches 75KLDjM4I1oi
Creating missing sync record for branches 7rqB6jvm3HxW
Creating missing sync record for branches 7wf8ETNNSP0f
Creating missing sync record for branches 8KHS9HGs7iDg
Creating missing sync record for branches 9ufJSnGz1mBs
Creating missing sync record for branches 9vEOJ6rSRfAn
Creating missing sync record for branches AGOsqqBjutFr
Creating missing sync record for branches AS0UdUFydkAk
Creating missing sync record for branches AWG2vJuxo1iR
Creating missing sync record for branches AkaRupITwAwH
Creating missing sync record for branches BXeV2bVIvxJz
Creating missing sync record for branches BlR0WIOVzmIa
Creating missing sync record for branches BqjqPlAl0Utf
Creating missing sync record for branches Ca2zH91QhZqO
Creating missing sync record for branches DHQ5IEuJQ3XJ
Creating missing sync record for branches DWbriyAzjYMi
Creating missing sync record for branches De5errANa3tr
Creating missing sync record for branches DgysjlxSsbVN
Creating missing sync record for branches F9vovplEsdDs
Creating missing sync record for branches FHgCDIofwrJt
Creating missing sync record for branches FtU9jKjEhqCc
Creating missing sync record for branches HPS8RJEaAT1p
Creating missing sync record for branches IIl0rg49hpz1
Creating missing sync record for branches JFcR72Lk5nsy
Creating missing sync record for branches MaRwpts7GeZl
Creating missing sync record for branches N7CRpy2HDLHe
Creating missing sync record for branches OH97tq3X9tv4
Creating missing sync record for branches OKqWgLlDOY3O
Creating missing sync record for branches OS53EqRfukt1
Creating missing sync record for branches OVLad732no3N
Creating missing sync record for branches PPimKh2OQMnM
Creating missing sync record for branches PXhtZlGWMoky
Creating missing sync record for branches Q2ITbug6IjJn
Creating missing sync record for branches QE2pPLvPJCwj
Creating missing sync record for branches QMEhWQWJ5fT3
Creating missing sync record for branches QeNmy6YrAkdp
Creating missing sync record for branches R6JXMJiFcxKf
Creating missing sync record for branches RVzer7O9ZnmR
Creating missing sync record for branches Rxgw9CDbmpC6
Creating missing sync record for branches S7ngIinIw0ze
Creating missing sync record for branches TBAzknehnA63
Creating missing sync record for branches TLQlzwRgarlz
Creating missing sync record for branches TOqJHit3NGQt
Creating missing sync record for branches TnTd2vR2MN2A
Creating missing sync record for branches UVj4yVOmeXo0
Creating missing sync record for branches UqXz7Ftzhe7X
Creating missing sync record for branches VJkcWy1VYGtk
Creating missing sync record for branches W6juRxokixlB
Creating missing sync record for branches WA7Y3xGeFegb
Creating missing sync record for branches WoVUHUwLiI48
Creating missing sync record for branches WsN82SLeiKfg
Creating missing sync record for branches Y9yGkHPBTvxX
Creating missing sync record for branches YaST8TdH9uRV
Creating missing sync record for branches Z8SmJd7nQWDj
Creating missing sync record for branches ZKt3DMBmidC7
Creating missing sync record for branches aJNnJQ3DMBRA
Creating missing sync record for branches aOR3m8SZjlfX
Creating missing sync record for branches afufw1BIVPmV
Creating missing sync record for branches b53zICbuDnGV
Creating missing sync record for branches e3rKRzeENb6k
Creating missing sync record for branches eCzXiZPosdT1
Creating missing sync record for branches fDOk5Ux9gAWl
Creating missing sync record for branches fFSE3qkoScUG
Creating missing sync record for branches fGDuwgO7mah3
Creating missing sync record for branches fRrCYmOf2sdm
Creating missing sync record for branches fnQWM6DGimHJ
Creating missing sync record for branches fnhRFDQKHB9Q
Creating missing sync record for branches fpmIhKdr42py
Creating missing sync record for branches gWJ63bEBxE5a
Creating missing sync record for branches gccufdCd1mJB
Creating missing sync record for branches hsoje9HMS5ql
Creating missing sync record for branches lTMylpwZ3Q41
Creating missing sync record for branches lWdI0ERF86Ze
Creating missing sync record for branches lvhjIyTQ0fpA
Creating missing sync record for branches m4wqtzFI6fe6
Creating missing sync record for branches mMXkjIXZm1v9
Creating missing sync record for branches mvCxIvCLYVL6
Creating missing sync record for branches ncw8v0IdmNJx
Creating missing sync record for branches nfNGjy5SGlnt
Creating missing sync record for branches nvDka7U4tkeL
Creating missing sync record for branches oAomPWRI1Spf
Creating missing sync record for branches oadWGhGMDCJ7
Creating missing sync record for branches oyjltcG4GaLZ
Creating missing sync record for branches plJ8iLegm3zK
Creating missing sync record for branches qUADXJnq2DGr
Creating missing sync record for branches qhpirQZDOxzB
Creating missing sync record for branches qsKENJTpzVoN
Creating missing sync record for branches rdQNZf4yNAlH
Creating missing sync record for branches root
Creating missing sync record for branches rrBVONtUUCB3
Creating missing sync record for branches tezCSMjhIdrV
Creating missing sync record for branches uqgDLmIoStdG
Creating missing sync record for branches vWVNL3eBUwpJ
Creating missing sync record for branches vvd1k4ESr5Zx
Creating missing sync record for branches wGzjoXePJIuA
Creating missing sync record for branches wicDo9ziq2n6
Creating missing sync record for branches wqXYybn6Zu4g
Creating missing sync record for branches wsmzd5JUa36D
Creating missing sync record for branches xT7kbOh4CjeN
Creating missing sync record for branches xp9q6q9sIlSk
Creating missing sync record for branches xuIjHon31Q0Q
Creating missing sync record for branches xwe34HlBkluB
Creating missing sync record for branches yIWeOguwXoLY
Creating missing sync record for branches yIxSdw6cpoRA
Creating missing sync record for branches yNDsTvUQ4ZDZ
Creating missing sync record for branches ytMsodGfOeBa
Trace: ERROR: Error executing query. Inner exception: Error: SQLITE_ERROR: no such column: noteIdError
    at wrap (/usr/src/app/src/services/sql.js:160:23)
    at Object.execute (/usr/src/app/src/services/sql.js:145:26)
    at cleanupSyncRowsForMissingEntities (/usr/src/app/src/services/sync_table.js:57:15)
    at fillSyncRows (/usr/src/app/src/services/sync_table.js:65:11)
    at Object.fillAllSyncRows (/usr/src/app/src/services/sync_table.js:94:11)
    at Object.error (/usr/src/app/src/services/log.js:33:13)
    at wrap (/usr/src/app/src/services/sql.js:166:13)
Trace: ERROR: Error executing transaction, executing rollback. Inner stack: Error
    at wrap (/usr/src/app/src/services/sql.js:160:23)
    at Object.execute (/usr/src/app/src/services/sql.js:145:26)
    at cleanupSyncRowsForMissingEntities (/usr/src/app/src/services/sync_table.js:57:15)
    at fillSyncRows (/usr/src/app/src/services/sync_table.js:65:11)
    at Object.fillAllSyncRows (/usr/src/app/src/services/sync_table.js:94:11)
Outside stack: Error
    at Object.transactional (/usr/src/app/src/services/sql.js:187:23)
    at cls.init (/usr/src/app/src/routes/routes.js:80:38)
    at /usr/src/app/node_modules/cls-hooked/context.js:118:13
    at Namespace.run (/usr/src/app/node_modules/cls-hooked/context.js:97:5)
    at Namespace.runAndReturn (/usr/src/app/node_modules/cls-hooked/context.js:117:8)
    at Object.init (/usr/src/app/src/services/cls.js:5:28)
    at router.(anonymous function) (/usr/src/app/src/routes/routes.js:75:38)
    at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:95:5)
    at next (/usr/src/app/node_modules/express/lib/router/route.js:137:13)
    at checkAppNotInitialized (/usr/src/app/src/services/auth.js:55:9)
    at Object.error (/usr/src/app/src/services/log.js:33:13)
    at Promise (/usr/src/app/src/services/sql.js:205:21)
Trace: ERROR: post /api/setup/new-document threw exception: Error
    at wrap (/usr/src/app/src/services/sql.js:160:23)
    at Object.execute (/usr/src/app/src/services/sql.js:145:26)
    at cleanupSyncRowsForMissingEntities (/usr/src/app/src/services/sync_table.js:57:15)
    at fillSyncRows (/usr/src/app/src/services/sync_table.js:65:11)
    at Object.fillAllSyncRows (/usr/src/app/src/services/sync_table.js:94:11)
    at Object.error (/usr/src/app/src/services/log.js:33:13)
    at router.(anonymous function) (/usr/src/app/src/routes/routes.js:94:17)

**Comment by zadam** [comment]
Thank you for reporting this bug, fix is in 0.32.1-beta release.

**Comment by 3dinfluence** [comment]
@zadam I just wanted to say that I saw this `no such column: noteIdError` when upgrading from `0.30.8` to `0.32.3`.  My fix was to install `0.31.6` and launch that version first which ran some schema changes.  After that I was able to successfully upgraded and launch `0.32.3`
